rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date)
```

Comment:
- Hey, let's modify our constructor for Rental so Book and Person are set in it to break the many-to-many relationship. 
  `def initialize(date, book, person)`